static-site/src/components/ListResources/types.ts

```diff
@@ -0,0 +1,45 @@
+// FIXME: is this in react-select?
+export type FilterOption = {
+    value: string
+    label: string
+}
+
+export type Group = {
+    groupName: any
```

Comment:
`any` types, IME, are timebombs. If you really can't type it any more precisely than this, use `unknown` -- that will force you to deal with things the compiler doesn't like. An `any` type is just a way of doing a lot of extra work to get zero benefit.

Reply:
Agreed. I put `any` for things that I couldn't deduce quickly. I'll take a closer look at each and consult @jameshadfield if I still can't figure it out.

Reply:
In the past, I've had decent luck looking at what VS Code (or any LSP-capable editor) infers based on where the type'd thing is being used — not 100% of the time, but it's sometimes a short cut.

Reply:
Yeah I did that where it was helpful - all the `any`s were already detected as `any` by VS Code. I've gone through most but there's a few left in 6198eef3a51c07fdf31778377afe68d80810b700. Might get to them tomorrow but I'm signing off for the day.

Reply:
Gradual adoption of TS to a codebase is a balancing act as judicial use of `any`/`unknown` is necessary to stop you introducing types to the whole codebase! But for this case, `groupName` is within the boundary of `ListResources` so it should be typed. Looking at 6198eef there's 7 `any` types left, and all of them should be typeable. I can help with this next week.

Reply:
Replaced explicit `any`s in 8e5e5daab7b489ef5368d881f45f11930ef07de7